readme.md

```diff
@@ -48,4 +48,11 @@ What you have to do in this phase is separate your original html page into 3 par
 2. table.pug: Show the original data and the rank corresponding to the items in the graph
 3. controllers/topologicalSort: Modify the showPath function and send the necessary data to the page by render. The const data is an array of objects which contains the original food data.
 
-* You can modify the css file as you want.
\ No newline at end of file
+* You can modify the css file as you want.
+###How to decide thresholds
+```
+threLower: The path will not include the point(health_value, taste_value) if a*health_value + b*teaste_value < threLower 
+threUpper: The path will connect the point(health_value, taste_value) on both sides(connect it twice) if a*health_value + b*teaste_value > threUpper
+For now, a = 0.5, b = 0.5, they represent the importance of taste and health.
+Usually, threLower can be 1 to 1.5, and threUpper can be 6 to 7. It depends on the distribution of input data.
+```
```

Comment:
Can you give out the example specify how the distribution uses threLower = 1, threUpper = 6, etc?

Reply:
It's hard to say. You can adjust them according to the result path. For example, the distribution of your data is hyperbolic, and the most of points concentrate on the origin of coordinates. If threLower is 1, the result will contain too many points with both low values, so threLower should be bigger. And about threUpper, if you can hardly find a point connected twice in a path, it should be smaller.

Reply:
Alright. Maybe you can add some sliders for thresholds and axis (**var points = (50*point1[0] + 20)..** in the pug file ) to help the user choose the value in the next PR.